exporter/earthlyoutputs/export.go

```diff
@@ -457,7 +457,7 @@ func (e *imageExporterInstance) Export(ctx context.Context, src *exporter.Source
 			md[safeGrpcMetaKey(mdK)] = string(mdV)
 		}
 		if expSrc.Ref == nil {
-			return nil, nil, fmt.Errorf("dirExpSrcs got nil ref")
+			return nil, nil, errors.Errorf("dirExpSrcs got nil ref")
```

Comment:
false negative by linter

```suggestion
			return nil, nil, errors.New("dirExpSrcs got nil ref")
```